manifests/postgresql-operator-default-configuration.yaml

```diff
@@ -62,41 +63,41 @@ configuration:
     master_dns_name_format: "{cluster}.{team}.{hostedzone}"
     replica_dns_name_format: "{cluster}-repl.{team}.{hostedzone}"
   aws_or_gcp:
-    # db_hosted_zone: ""
-    # wal_s3_bucket: ""
-    # log_s3_bucket: ""
-    # kube_iam_role: ""
-    aws_region: eu-central-1
     # additional_secret_mount: "some-secret-name"
     # additional_secret_mount_path: "/some/dir"
+    aws_region: eu-central-1
+    # kube_iam_role: ""
+    # log_s3_bucket: ""
+    # wal_s3_bucket: ""
+  logical_backup:
+    logical_backup_schedule: "30 00 * * *"
+    logical_backup_docker_image: "registry.opensource.zalan.do/acid/logical-backup"
+    logical_backup_s3_bucket: ""
```

Comment:
there is no default for that, people have to add their own, that is `logical_backup_s3_bucket: "your-bucket"`